Sasquatch/SasquatchSwift/AppDelegate.swift

```diff
@@ -212,18 +212,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CrashesDelegate, CLLocati
   // Crashes Delegate
 
   func crashes(_ crashes: Crashes, shouldProcess errorReport: ErrorReport) -> Bool {
-
-    // return true if the crash report should be processed, otherwise false.
+    if errorReport.exceptionReason != nil {
+      NSLog("Should process error report with: %@", errorReport.description);
+    }
     return true
   }
 
   func crashes(_ crashes: Crashes!, willSend errorReport: ErrorReport!) {
+    if errorReport.exceptionReason != nil {
+      NSLog("Will send error report with: %@", errorReport.description);
```

Comment:
```suggestion
      NSLog("Will send error report: %@", errorReport.description);
```

Reply:
Please pay attention that all these logs were copied from other our test apps, let's keep it as it is.